### ISSUE TITLE
BadRequest: A transition is required

### ISSUE THREAD

**Issue by deiferni** [4teamwork/opengever.core#4831]
When triggering task transitions the form offers a field with which to add references (a.k.a. Verweise). When i attempt to browse for said refereces this triggers the following error however:

https://sentry.4teamwork.ch/sentry/onegov-gever/issues/8853/

```
BadRequest: A transition is required
(3 additional frame(s) were not displayed)
...
  File "apps/eggs/zope.traversing-3.13.2-py2.7.egg/zope/traversing/namespace.py", line 112, in namespaceLookup
    return traverser.traverse(name, ())
  File "apps/eggs/plone.z3cform-0.8.1-py2.7.egg/plone/z3cform/traversal.py", line 60, in traverse
    form.update()
  File "apps/01-dev.onegovgever.ch-fd/src/opengever.core/opengever/task/response.py", line 202, in update
    if self.is_final_transition:
  File "apps/01-dev.onegovgever.ch-fd/src/opengever.core/opengever/task/response.py", line 226, in is_final_transition
    return self.transition in FINAL_TRANSITIONS
  File "apps/01-dev.onegovgever.ch-fd/src/opengever.core/opengever/task/response.py", line 221, in transition
    raise BadRequest("A transition is required")

BadRequest: A transition is required
```

**Comment by phgross** [comment]
Happens during unauthorized traversal request, will be fixed with #4786

**Comment by deiferni** [comment]
so we close this one?
